README.md

```diff
@@ -1,36 +1,48 @@
-#WaveLoading
-
-This library provides a *wave loading* animation as a **Drawable**.
+# WaveLoading
+  WaveLoading: This library provides a wave loading animation as a Drawable Element.
 
 ![screenshot](./screenshots/screenshot.gif)
+ 
+## Usage Instructions
```

Comment:
Use HarmonyOS images and avoid using Android images

Reply:
Removed android images, added an apple image.